### PR TITLE
Add find and sort to DG and fix UG bugs

### PR DESCRIPTION
**Pull request by Nanosync**
- Add find command to DG
- Add sort command use case to DG
- Fixes `update` to `edit` in UG
- Cleanup some spelling mistakes

#92

### CODE REVIEW

File: docs/diagrams/FindActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@startuml
+start
+:User executes find command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if () then ([valid fields present])
```
Comment (minpyaemoe) [inline docs/diagrams/FindActivityDiagram.puml:8]: i think there should be two type of errors, right? like missing fields and invalid fields?

Comment (Nanosync) [reply docs/diagrams/FindActivityDiagram.puml:8]: The sequence diagram treats all these cases as `else`.

**Review by choongyx — APPROVED** [review]

**Comment by Nanosync** [comment]
Coverage drop is unrelated

File: docs/diagrams/FindActivityDiagram.puml (hunk 1 above)
Comment (minpyaemoe) [reply docs/diagrams/FindActivityDiagram.puml:8]: But have different error messages, right? no?

**Pull request merged**